scripts/bundleSize/createConsoleError.js

```diff
@@ -0,0 +1,10 @@
+const chalk = require('chalk');
+
+module.exports = ({ service, pageType, size, adjective }) =>
+  [
+    chalk.red('Bundle size for'),
+    chalk.red.bold(`${service} ${pageType}`),
+    chalk.red(`is too ${adjective} at`),
+    chalk.red.bold(`${size} kB.`),
+    chalk.red("Please update thresholds in './scripts/bundleSizeConfig.js'"),
```

Comment:
```suggestion
    chalk.red("Please update thresholds in './scripts/bundleSize/bundleSizeConfig.js'"),
```
I noticed the bundleSizeConfig.js was moved. The unit tests will have to be updated also.

Reply:
Thanks @thekp, I'll take a look

Reply:
@thekp - should be resolved by https://github.com/bbc/simorgh/pull/7229/commits/9314b84b4567712f61b74aea6499457442d7bca3